src/frontend/list.js

```diff
@@ -0,0 +1,40 @@
+const createLi = (user, score) => {
+  const li = document.createElement('li');
+  const p = document.createElement('p');
+  p.innerText = `${user}: ${score}`;
+  li.appendChild(p);
+
+  return li;
+};
+
+const refreshList = (objList) => {
+  const ul = document.getElementById('ul-list');
+  ul.innerHTML = '';
+
+  objList.forEach((obj) => ul.appendChild(createLi(obj.user, obj.score)));
+};
+
+const setRefreshBtn = (api) => {
+  const btn = document.getElementById('refresh-btn');
+  btn.addEventListener('click', () => {
+    api.getList();
+  });
+};
+
+const addToList = (user, score) => {
+  const ul = document.getElementById('ul-list');
+  ul.appendChild(createLi(user, score));
+};
+
+const setAddBtn = (api) => {
+  const enterName = document.getElementById('enter-name');
+  const enterScore = document.getElementById('enter-score');
+  const addBtn = document.getElementById('add-btn');
```

Comment:
## * [OPTIONAL]
* I think you can clear the input fields after adding a new score this would improve user experience 💯